xrpl_api/src/types/transaction/variants/account_delete.rs

```diff
@@ -1,12 +1,38 @@
 use crate::TransactionCommon;
+use enumflags2::BitFlags;
 use serde::{Deserialize, Serialize};
+use xrpl_types::AccountDeleteFlags;
 
 /// An `AccountDelete` transaction <https://xrpl.org/accountdelete.html>
 #[derive(Debug, Clone, Deserialize, Serialize)]
 #[serde(rename_all = "PascalCase")]
 pub struct AccountDeleteTransaction {
     #[serde(flatten)]
     pub common: TransactionCommon,
+    #[serde(default)]
+    pub flags: BitFlags<AccountDeleteFlags>,
```

Comment:
Added flags to all transaction variants that uses the bitflags defined in `xrpl_types` "binary" transactions